docs/config.ld

```diff
@@ -0,0 +1,33 @@
+project = 'ATF'
+title = 'ATF SW Detail Design documentation'
+description = [[
+# Automated Test Framework (ATF) system
+ATF is a C++/Lua framework for SDL automated black-box testing.
+It provides high- and low-level API for mobile applications and/or HMIs emulation.
+
+For High-level information, please follow [ATF SW Architecture Document](https://smartdevicelink.com/en/guides/pull_request/93dee199f30303b4b26ec9a852c1f5261ff0735d/atf/table-of-contents/)
+
+For Detail design information, please follow related SDD.
+
+## LICENSE
+The code is copyright [Ford Motor Company](https://github.com/smartdevicelink/sdl_core/blob/master/LICENSE).
```

Comment:
@LuxoftAKutsan,
We have a FORD license. How we can avoid mentioning it?
Do you have any other suggestions?